src/RiotAPILibraries.ts

```diff
@@ -50,6 +50,13 @@ interface LibraryDescription {
 }
 
 export default class RiotAPILibraries {
+    static CHANNEL_TOPICS = {
+        lol: ["lol", "v4"],
+        lcu: ["lcu"],
+        tft: ["tft"],
+        lor: ["lor"]
+    } as const;
+
```

Comment:
Maybe consider moving this to shared_settings.json

Reply:
good catch 👍